recipes/smurff/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set version = "0.17.0" %}
+
+package:
+  name: smurff
+  version: {{ version }}
+
+source:
+  url: https://github.com/ExaScience/smurff/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 998bb056b76c87feba4af2f1692236ffa420cc547f838a93262c8b846b8dea5d 
+
+build:
+    number: 0
+    script_env:
+      - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - mkl-devel
+    - eigen
+    - libboost
+    - pybind11
+    - highfive >=2.2
+    - h5py
+    - python {{ python }}
+    - setuptools
+    - setuptools_scm
+    - numpy
+  run:
+    - python {{ python }}
+    - mkl
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - libboost     # [not win]
```

These should be added by their `run_exports` and `boost-cpp` is the correct name here.
```suggestion
  - boost-cpp
```